measurements/geography_based.py

```diff
@@ -40,6 +45,11 @@ def country_to_iso3(country):
     return iso3
 
 def geo_ctr(dataloader, args):
+    # redirect to geo_ctr_gps if dataset is of gps form:
+    if (dataloader.dataset.geo_boundaries is not None):
+        print("redirecting to geo_ctr_gps()...")
+        return geo_ctr_gps(dataloader, args)
+    print("starting geo_ctr() original...")
```

Comment:
geo_ctr "original" doesn't mean anything -- gps vs country labels, neither is more "original" than the other, so just call them by names